packages/editor-ui/src/components/CodeNodeEditor/CodeNodeEditor.vue

```diff
@@ -133,6 +127,7 @@ export default mixins(linterExtension, completerExtension, workflowHelpers).exte
 					insertedText = full.slice(lastDotIndex + 1);
 				}
 
+				// TODO: Still has to get updated for Python and JSON
```

Comment:
does this need to be addressed still?

Reply:
Yes and no. Currently there is only autocomplete for JS and not Python. Meaning right now it is fine like that but once autocomplete does get added also for Python product is probably interested in knowing if it is JS or Python.